.github/labeler.yml

```diff
@@ -0,0 +1,18 @@
+client:
+  - client/flutter/**
```

Comment:
> 
```suggestion
  - client/**
```

Reply:
@advayDev1 I've been using the `client` tag to refer to the Flutter client to make it easier to direct people to Flutter tasks/PRs to review. Happy to rename `client` to `flutter` if you think that's better, or we can include other clients in the `client` tag

Reply:
Current operating assumption is just 1 client right? So imo just use client.

Reply:
the question is if we want changes in the [`client/rpc`](https://github.com/WorldHealthOrganization/app/tree/master/client/rpc) directory also labeled with the `client` tag

Reply:
I will accept this change as proposed for now. We don't have a high volume of PRs editing the `client/rpc` directory anyway, so probably not worth discussing further at this point... if that does become an issue, we can always edit this later